.github/workflows/build-test-release.yml

```diff
@@ -12,6 +12,10 @@ on:
       - "stable/**"
     tags:
       - 'v*'
+  pull_request:
```

Comment:
Should amend the comment above (line 8).

Is this also very indirectly the reason for the coverage dropping to 0 for all C++ files? This is also part of the other PR you just opened where the coverage also drops.

Reply:
Oh thats a second commit for testing only, I'll remove it before merging.